disnake/ext/commands/common_bot_base.py

```diff
@@ -82,13 +81,6 @@ def __init__(
 
         super().__init__(*args, **kwargs)
 
-    def dispatch(self, event_name: str, *args: Any, **kwargs: Any) -> None:
-        # super() will resolve to Client
-        super().dispatch(event_name, *args, **kwargs)  # type: ignore
```

Comment:
```suggestion
    # FIXME: make event name pos-only or remove entirely in v3.0
    def dispatch(self, event_name: str, *args: Any, **kwargs: Any) -> None:
        # super() will resolve to Client
        super().dispatch(event_name, *args, **kwargs)  # type: ignore
```
The event parameter names differ (`event_name` here vs `event` in client), so this method should stay a little longer for now - `bot.dispatch(event_name="stuff")` would no longer work otherwise.
Might be overly cautious, but it also has the nice benefit of avoiding a few type ignores in other places as well.